lib/credentials/ecs_credentials.d.ts

```diff
@@ -1,11 +1,11 @@
-import {Credentials} from '../credentials';
-export class ECSCredentials extends Credentials {
-		/**
-		 * Represents credentials received from relative URI specified in the ECS container.
-		 * @param {object} options - Override the default (1s) timeout period.
-		 */
-		constructor(options?: ECSCredentialsOptions);
-	}
+import {RemoteCredentials} from '../remote_credentials';
```

Comment:
`remote_credentials` is located in the same directory, so it should be `./remote_credentials`

Reply:
Ah of course. I added the fix and all tests appear to be passing now. Thx